src/data/data_pool.py

```diff
@@ -180,6 +180,9 @@ def get_format_list(self):
     def get_comment_sign(self):
         return self.comment_sign
 
+    def get_fgen(self):
```

Comment:
Not really necessary since `.fgen` can be accessed directly. If you need getters / setters to protect against arbitrary assignment, take a look at: https://docs.python.org/2/library/functions.html#property.